challenge_6/scala/ndsmith3/Ranges.scala

```diff
@@ -0,0 +1,34 @@
+import scala.annotation.tailrec
+
+object Ranges {
+  @tailrec
+  def makeRangeString(answer: List[String])(curr: Array[Int]): List[String] = {
+    val answerPlusNext       = (curr.head.toString :: answer)
+    lazy val nextIsIncrement = curr.head + 1 != curr(1)
```

Comment:
Not sure the lazy variables here really buy you much, but their overhead should be negligible anyway.